brayns/Brayns.cpp

```diff
@@ -115,49 +123,90 @@ struct Brayns::Impl
         if (!_engine)
             throw std::runtime_error("Unsupported engine: " + engineName);
 
+        _engine->getFrameBuffer().clear();
```

Comment:
the frameBuffer isn't cleared initially?

Reply:
not explicitly in Brayns, ospray and optix might do it.